reactioncommerce/catalyst
# chore: initial setup of Catalyst

# Issue

Need a home for the Catalyst components.

# Solution

Create a new repo based on https://github.com/reactioncommerce/reaction-component-library. Sharing the same repo prooved too cumbersome.

# Testing

1. Clone, then copy `.env.example` -> `.env`
1. Run the docs with `docker-compose up` from the root directory
1. Open docs at http://localhost:4060, or at the port you set in .env
1.  Check that the doc renders and that you see the `Button` component page with minimal content.
1. Run the tests with `docker-compose run --rm web yarn test` and ensure they pass 
1. Check the circle-ci, docker, package.json and other places to ensure that this package will be built/published as `@reactioncommerce/catalyst` and **not** `@reactioncommerce/components` on npm.
